components/card/card.css

```diff
@@ -65,19 +65,47 @@
 .card-container-portrait {
   display: flex;
   flex-direction: column;
+  width: 15rem;
   background-color: var(--white-color);
   padding: 1rem;
   border-radius: 0.7rem;
   gap: 10px;
+  box-shadow: 1px 1px 4px #888888;
 }
-.image-container {
+.card-image-container {
   position: relative;
+  align-self: center;
+}
+.card-container-textonly {
+  background-color: var(--white-color);
+  width: 15rem;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  border-radius: 1rem;
+  box-shadow: 1px 1px 4px #888888;
+}
+.card-container-vertical {
+  background-color: var(--white-color);
+  width: 20rem;
+  padding: 1rem;
+  display: flex;
+  gap: 1rem;
+  border-radius: 1rem;
+  box-shadow: 1px 1px 4px #888888;
```

Comment:
You can use a variable here too. `#888888` is used multiple times

Reply:
yes done